src/common/MobileOnboarding/AppMobileOnboarding.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```

Comment:
this component shouldn't bee in `common` as it's component filled with end-content